### PR TITLE
Run linter on all files

### LINKED ISSUE #35: Update and run lintr
Lintr should include semi-colon (and quotations if possible) and should be run on files

### PR DESCRIPTION
**Pull request by alia-peterson**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [ ] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [x] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
Added semi-colons to linter and ran linter fix 

## *Instructions for reviewer:*
Most if not all js files have minor adjustments. Double check that nothing broke in website.
*What are the relevant tickets?*
Closes #35

## *Issues:*
No known issues

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -7,7 +7,10 @@
     "jquery": true
   },
   "extends": "eslint:recommended",
-  "parserOptions": { "sourceType": "module" },
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaVersion": 2017
```
Comment (tashiad) [inline .eslintrc:12]: What was the reason for adding this?

File: src/scripts.js [hunk 2]
```diff
@@ -110,7 +110,7 @@ function displayPantryInfo(currentUser) {
   domUpdates.addPantryInfoToDom(currentUser.pantry)
 }
 
-async function updateUserPantryFromRecipe(recipeId, typeModification) {
```
Comment (tashiad) [inline src/scripts.js:113]: Why didn't this need to be async anymore?

**Review by tashiad — APPROVED** [review]
Looks good, just left a couple of questions!

**Review by richardltyler — APPROVED** [review]
Looks good to me! I second Tashia's question about removing that async and await function

**Pull request merged**